_data/toc.yaml

```diff
@@ -1583,6 +1577,52 @@ manuals:
           title: Install Buildx
         - path: /build/buildx/multiple-builders/
           title: Using multiple builders
+    - sectiontitle: Continuous integration
+      section:
+        - path: /build/ci/
+          title: CI with Docker
+        - sectiontitle: GitHub Actions
+          section:
+          - path: /build/ci/github-actions/
+            title: Docker GitHub Actions
+          - path: /build/ci/github-actions/gha-tutorial/
+            title: Tutorial
+          - sectiontitle: Advanced examples
+            section:
+            - path: /build/ci/github-actions/examples/append-nodes/
```

Comment:
As we're adding a new level; should we add an `index.md` inside `examples/` ? I see the breadcrumbs doesn't have an entry for it, but users may still try to traverse the URL (`/build/ci/github-actions/examples/append-nodes/` -> `/build/ci/github-actions/examples/`) and end up on a 404.

Reply:
There are quite a few entries on this level, so I avoided adding an overview page to reduce the clutter. I think I would prefer it like this—if users try to manually find a page that isn't linked anywhere by editing the path, maybe that's a valid 404.